recipes/r-paco/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set version = '0.3.2' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+{% set cran_mirror = 'https://cran.r-project.org' %}
+
+package:
+  name: r-paco
+  version: {{ version|replace("-", "_") }}
+
+source:
+  fn: paco_{{ version }}.tar.gz
+  url:
+    - https://cloud.r-project.org/src/contrib/paco_{{ version }}.tar.gz
+    - https://cloud.r-project.org/src/contrib/Archive/paco/paco_{{ version }}.tar.gz
+  sha256: cc40096d0803fa07e7b7c7e4808dbfbac8cab5e29e529f203c0d608a9d48c661
+
+build:
+  number: 0
+  skip: true  # [win32]
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+  host:
+    - r-base
+    - r-plyr
+    - r-vegan >=2.2_0
+  run:
+    - r-base
+    - r-plyr
+    - r-vegan >=2.2_0
+
+test:
+  commands:
+    - $R -e "library('paco')"           # [not win]
+    - "\"%R%\" -e \"library('paco')\""  # [win]
+
+about:
+  home: http://www.uv.es/cophylpaco/
+  license: MIT
+  summary: Procrustes analyses to infer co-phylogenetic matching between pairs of (ultrametric)
+    phylogenetic trees.
+  license_family: MIT
+
+extra:
+  recipe-maintainers:
+    - nick-youngblut
```

Comment:
Can you please add:
```
    - johanneskoester
    - bgruening
    - daler
    - jdblischak
   - cbrueffer
```

here as well? was this not added by the script?

Reply:
Sorry! I read the docs wrong, and so I thought that I was supposed to be the sole maintainer.